src/creatures/creature.cpp

```diff
@@ -731,6 +729,7 @@ bool Creature::dropCorpse(Creature* lastHitCreature, Creature* mostDamageCreatur
 
 		if (corpse) {
 			dropLoot(corpse->getContainer(), lastHitCreature);
+			corpse->startDecaying();
```

Comment:
any reason this if and the other if are separated, can't we unify the if (corpse)?

Reply:
Done!